index.html

```diff
@@ -0,0 +1,75 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=  device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header class="first-header">
+        Technical Documentation
+        </header>
+        <ul>
+            <li><a href="#The_first_section" class="nav-link">The first section</a></li>
+            <li><a href="#Introduction" class="nav-link">Introduction</a></li>
+            <li><a href="#What_you_Should_know" class="nav-link">What you should already know?</a></li>
+            <li><a href="#what_is_Javascript_and_Java?" class="nav-link">What is Javascript and Java?</a></li>
+            <li><a href="#Hello_World!" class="nav-link">Hello World!</a></li>
+        </ul>
+    </nav>
+    <main id="main-doc">
+        <div class="SecondContainer">
+            <section class="main-section" id="The_first_section">
+                <header>The first section </header>
```

Comment:
I think you shouldn't use 'header' tags for titles inside the section tags. instead, you should use the 'h3 or h4' tags depending on the size you want to apply for the title.

Reply:
OPTIONAL: Well-written review comment :clap:. Kindly make sure you have no grammatical or typo errors. I saw a word that is not initialized after the full stop. :pray: